package.json

```diff
@@ -15,6 +15,7 @@
     "type-check": "tsc --noEmit",
     "test": "yarn lint && yarn type-check",
     "test:unit": "mocha -r ts-node/register --timeout 999999 ./test/**/*test.ts",
+    "test:single": "mocha -r ts-node/register --timeout 999999",
```

Comment:
What are we testing here?

Reply:
Just added that for single file testing and other options but will remove that as we do not need that now.